tests/PHPUnit/Builder/KataBuilderTest.php

```diff
@@ -0,0 +1,109 @@
+<?php
+/**
+ * This file is part of the phpkata project.
+ *
+ * (c) Yannick Voyer (http://github.com/yvoyer)
+ */
+
+namespace Star\Kata\Builder;
+
+use Star\Kata\Model\Kata;
+
+/**
+ * Class KataBuilderTest
+ *
+ * @author  Yannick Voyer (http://github.com/yvoyer)
+ *
+ * @package Star\Kata\Builder
+ *
+ * @covers Star\Kata\Builder\KataBuilder
+ * @uses Star\Kata\Builder\ObjectiveBuilder
+ * @uses Star\Kata\Model\Objective\ConfigurableObjective
+ * @uses Star\Kata\Model\Kata
+ */
+final class KataBuilderTest extends \PHPUnit_Framework_TestCase
```

Comment:
I'm planning to use behat. 

For phpspec, i'm just not confortable with it still. 

You welcome to suggest a pr that would include phpunit equivalent tests.